pkg/test/params.go

```diff
@@ -22,11 +22,11 @@ import (
 )
 
 type Params struct {
-	ns, kc string
-	Tekton versioned.Interface
-	Kube   k8s.Interface
-	Clock  clockwork.Clock
-	Cls    *cli.Clients
+	ns, kc, c string
```

Comment:
`c` doesn't make it obvious what it is imho. How about kContext?

Reply:
and make kc -> kConfig

Reply:
I have updated `c` to `kContext` and `kc` to `kConfig`.